src/lib/accessibility/Behaviors/Icon/iconBehavior.ts

```diff
@@ -2,8 +2,9 @@ import { Accessibility } from '../../types'
 
 /**
  * @description
- * Adds attribute 'aria-hidden=true' to icon.
  * Icon is usually only visual representation and therefore is hidden from screen readers.
+ * @specification
```

Comment:
nit: could we, please, just for the sake of it being visually easier to read, split these sections by empty line?

Reply:
new line added :)